src/v_video.c

```diff
@@ -189,6 +191,17 @@ void V_InitColorTranslation(void)
 
   v_lightest_color = I_GetPaletteIndex(playpal, 0xFF, 0xFF, 0xFF);
   v_darkest_color  = I_GetPaletteIndex(playpal, 0x00, 0x00, 0x00);
+
+  byte *palsrc = playpal;
+  for (int i = 0; i < 256; ++i)
+  {
+    double red   = *palsrc++ / 256.0;
+    double green = *palsrc++ / 256.0;
+    double blue  = *palsrc++ / 256.0;
+
+    int gray = (red * 0.299 + green * 0.587 + blue * 0.144) * 255;
```

Comment:
So, "gray" is basically "vanilla" but without the inversion?

Reply:
Yes, correct.